README.md

```diff
@@ -44,41 +44,41 @@ View video demo of deploy goal-oriented bot and slot-filling model with Telegram
  ```
  python deep.py interact configs/go_bot/gobot_dstc2.json
  ```
- * Run slot-filling model with Telegram interface
+ * Run slot-filling model with Telegram interface:
  ```
  python deep.py interactbot configs/ner/slotfill_dstc2.json -t <TELEGRAM_TOKEN>
  ```
- * Run slot-filling model with console interface
+ * Run slot-filling model with console interface:
  ```
  python deep.py interact configs/ner/slotfill_dstc2.json
  ```
 ## Conceptual overview
 
 ### Principles
-The library is designed following the principles:
- * end-to-end deep learning architecture as long-term goal
+The library is designed according to the following principles:
+ * end-to-end deep learning architecture as a long-term goal
  * hybrid ML/DL/Rule-based architecture as a current approach
  * modular dialog system architecture
- * component-based software engineering, reusability maximization
- * easy to extend and benchmark
- * multiple components by one NLP task with data-driven selection of suitable components
+ * component-based software engineering, maximization of reusability
+ * easy extension and benchmarking
+ * multiple solutions for one NLP task for flexible data-driven configuration
 
 ### Target Architecture
 Target architecture of our library:
 <p align="left">
 <img src="http://lnsigo.mipt.ru/export/images/deeppavlov_architecture.png" width="50%" height="50%"/>
 </p>
-DeepPavlov is built on top of machine learning frameworks (TensorFlow, Keras). Other external libraries can be used to build basic components.
```

Comment:
Why?

Reply:
Why what?

Reply:
`is built on top of machine learning frameworks` → `is built on top machine learning frameworks`